dasharo-compatibility/dcu.robot

```diff
@@ -51,16 +50,15 @@ DCU002.001 Change the serial number
     [Documentation]    This test case verifies that the serial number encoded
     ...    in the DMI table of an image can be changed using DCU.
     Skip If    not ${DCU_SERIAL_SUPPORT}    DCU002.001 not supported
-    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DCU002.001 not supported
 
     ${serial_no}=    Random Int    min=10000000    max=99999999
-    ${result}=    Run Process    bash    -c    cd ./dcu; ./dcuc smbios -s ${serial_no} ./coreboot.rom
+    ${result}=    Execute Command In Terminal    cd ./dcu; ./dcuc smbios -s ${serial_no} ./coreboot.rom; cd ..;
     Log    ${result.stdout}
     Log    ${result.stderr}
     Should Contain    ${result.stdout}    Success
     Flash Firmware    ./dcu/coreboot.rom
 
-    Power On
```

Comment:
Will fail if Power On does not actually restart the device, the firmware will not be reloaded